Testing/SystemTests/scripts/CMakeLists.txt

```diff
@@ -1,29 +1,24 @@
-# Write out a files containing 
-# - the data search directories
-# - the default save directory
-# These will be read by the MantidConfig class to configure the properties file
+# Write out a files containing - the data search directories - the default save
+# directory These will be read by the MantidConfig class to configure the
+# properties file
 
 # Data search directories
 set(DATA_DIRS "")
 # Input data
 set(SYSTEM_TEST_DATA_DIR ${ExternalData_BINARY_ROOT}/Testing/Data/SystemTest)
 list(APPEND DATA_DIRS ${SYSTEM_TEST_DATA_DIR})
-list(APPEND DATA_DIRS ${SYSTEM_TEST_DATA_DIR}/LOQ)
-list(APPEND DATA_DIRS ${SYSTEM_TEST_DATA_DIR}/SANS2D)
-list(APPEND DATA_DIRS ${SYSTEM_TEST_DATA_DIR}/LARMOR)
 list(APPEND DATA_DIRS ${SYSTEM_TEST_DATA_DIR}/INTER)
 # Reference results
-list(
-  APPEND
-    DATA_DIRS
-    ${ExternalData_BINARY_ROOT}/Testing/SystemTests/tests/framework/reference)
+list(APPEND DATA_DIRS
+     ${ExternalData_BINARY_ROOT}/Testing/SystemTests/tests/framework/reference
+)
 # Unit tests
 list(APPEND DATA_DIRS ${ExternalData_BINARY_ROOT}/Testing/Data/UnitTest)
 # Doc tests
 list(APPEND DATA_DIRS ${ExternalData_BINARY_ROOT}/Testing/Data/DocTest)
 
-# Output to file 
-# If this is changed, make sure to update the .gitignore in this directory
+# Output to file If this is changed, make sure to update the .gitignore in this
```

Comment:
Yeah the auto-formatting tends to work well on CMakeLists, but not in this case since it breaks down our bullet list comments. I've reverted it thanks :+1: